dds/DCPS/RTPS/BaseMessageUtils.h

```diff
@@ -222,6 +222,36 @@ inline RTPS::SequenceNumber_t to_rtps_seqnum(const DCPS::SequenceNumber& opendds
   return rtps_seqnum;
 }
 
+inline void append_submessage(RTPS::Message& message, const RTPS::Submessage& submessage) {
```

Comment:
Somewhere we have a generic `push_back`-style adapter for IDL sequences.  This may not be needed.